Resources/Server Info/Guidebook/Chef/sandvich.xml

```diff
@@ -55,6 +55,30 @@
 <GuideEntityEmbed Entity="FoodBreadBanana" Caption=""/>
 </Box>
 
+<Box Orientation="Vertical">
+	[color=#a4885c]Tofu Bread[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Dough
+- Tofu block
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="20 sec"/>
+<GuideEntityEmbed Entity="FoodBreadTofu" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Garlic Bread Slice[/color]
```

Comment:
Making a note here: I will add a full Garlic Bread loaf, so this is more of a placeholder for now.